tests/testthat/test-plot_seromodel.R

```diff
@@ -0,0 +1,313 @@
+library(testthat)
+library(ggplot2)
+library(rlang)
+library(purrr)
+
+# NOTE: If the variable `expected_plot` needs to be updated because of changes in the plots
+# You can use dput(extract_plot_data(plot)) to obtain the current structure of the plot
+
+# Common data
+data(veev2012)
+serosurvey <- veev2012
+
```

Comment:
Since the model you're using is the same for all tests, I think you should move it to this section of the code and run it only once.
```r
seromodel <- fit_seromodel(
serosurvey = serosurvey
)
```